components/process.rst

```diff
@@ -68,6 +83,10 @@ with a non-zero code)::
         echo $e->getMessage();
     }
 
+.. versionadded:: 3.1
+    Streaming the input or the output of a process using iterators
+    were added in Symfony 3.1
```

Comment:
I think we should split this.

Above the paragraph added above:

```rst
.. versionadded:: 3.1
    Support for streaming the output using iterators was introduced
    in Symfony 3.1.
```

After the "Streaming to the standard input of a Process" headline:

```rst
    Support for streaming the input of a process was introduced in
    Symfony 3.1.
```